images/tomcat/configs/latest.apko.yaml

```diff
@@ -0,0 +1,69 @@
+contents:
+  keyring:
+    - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+  repositories:
+    - https://packages.wolfi.dev/os
+  packages:
+    - wolfi-baselayout
+    - ca-certificates-bundle
+    - openjdk-17-jre
+    - tomcat
+    - tomcat-native
+    - busybox
+
+work-dir: /usr/share/tomcat/webapps
+
+environment:
+  JAVA_HOME: /usr/lib/jvm/java-17-openjdk
+  CATALINA_HOME: /usr/share/tomcat
+  PATH: /usr/sbin:/sbin:/usr/bin:/bin:/usr/share/tomcat/bin
+
+accounts:
+  groups:
+    - groupname: nonroot
+      gid: 65532
+  users:
+    - username: nonroot
+      uid: 65532
+      gid: 65532
+  run-as: nonroot
+
+paths:
+  - path: /usr/share/tomcat/conf
+    type: directory
+    permissions: 0o755
+    uid: 65532
+    gid: 65532
+  - path: /usr/share/tomcat/logs
```

Comment:
Tomcat writes to logs , can we redirect it to stdout ?

Reply:
As far as I can tell it looks like the expectation for Kubernetes users is to run a sidecar like fluent.

It does look like this is [possible](https://medium.com/codex/configuring-tomcat-logging-for-a-read-only-file-system-in-kubernetes-965556e55045) though.

I'm waiting to hear back from the customer on what they expect but should this image still switch to logging to stdout?